src/main/kotlin/com/hyperdevs/poeditor/gradle/PoEditorPlugin.kt

```diff
@@ -54,6 +54,7 @@ class PoEditorPlugin : Plugin<Project> {
                 defaultResPath.convention(mainResourceDirectory.asFile.absolutePath)
                 tags.convention(emptyList())
                 languageValuesOverridePathMap.convention(emptyMap())
+                minimumTranslationPercentage.convention(null)
```

Comment:
I'd set `-1` as the default value here, that way you can skip making the `minimumTranslationPercentage` `null` in other places.

Reply:
OK, I'll do that. 
Btw I was hesitant to use -1, as it's kind of making some assumptions (even though safe enough). I thought handling the null would be the cleanest / most reliable solution. I used -1 in that line `percentage < minimumTranslationPercentage ?: -1` just because it was way shorter to write 😀)